.github/workflows/dispatch-sim-evaluation-tools.yaml

```
@@ -0,0 +1,27 @@
+name: dispatch-sim-evaluation-tools
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  make_dispatch_event:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'execute-autoware-evaluator'
+    steps:
+      - name: generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
```

ここで発行しているtokenがtier4/autoware.universeのみで有効なものになっているようです
(github-app-tokenのworkflow的には正しい挙動)
ここではtokenをworkflow dispatchのみに使っているので，[README](https://github.com/tibdex/github-app-token/blob/main/README.md)に従ってこうすれば良さそうです
```suggestion
          private_key: ${{ secrets.PRIVATE_KEY }}
          repository: tier4/sim_evaluation_tools_test
```